doc/src/manual/code-loading.md

```diff
@@ -19,7 +19,7 @@ Understanding how Julia answers these questions is key to understanding package
 
 ## Federation of packages
 
-Julia supports federated management of packages. This means that multiple independent parties can maintain both public and private packages and registries of them, and that projects can depend on a mix of public and private packages from different registries. Packages from various registries are installed and managed using a common set of tools and workflows. The Pkg3 next-generation package manager [[docs](https://julialang.org/Pkg3.jl/latest/), [repo](https://github.com/JuliaLang/Pkg3.jl)] ships with Julia 0.7/1.0 and lets you install and manage dependencies of your projects, by creating and manipulating project files, which describe what your project depends on, and manifest files that snapshot exact versions of your project's complete dependency graph.
+Julia supports federated management of packages. This means that multiple independent parties can maintain both public and private packages and registries of them, and that projects can depend on a mix of public and private packages from different registries. Packages from various registries are installed and managed using a common set of tools and workflows. The Pkg next-generation package manager [[docs](https://julialang.org/Pkg.jl/latest/), [repo](https://github.com/JuliaLang/Pkg.jl)] ships with Julia 0.7/1.0 and lets you install and manage dependencies of your projects, by creating and manipulating project files, which describe what your project depends on, and manifest files that snapshot exact versions of your project's complete dependency graph.
```

Comment:
I don't think you want to change these since these links are now broken. Unless you're planning on moving these repos?

Reply:
I thought we would move the repo as well but can keep them until that has happened.